modules/microshift-certificate-lifetime.adoc

```diff
@@ -17,24 +17,41 @@ An example of a long-lived certificate is the client certificate for `system:adm
 
 [id="microshift-certificate-rotation_{context}"]
 == Certificate rotation
-As certificates age, {product-title} can be restarted to rotate certificates. A certificate that is close to expiring might also automatically cause a restart. Read the following situation overviews to understand the actions at each moment in time:
-
-. Green zone:
-.. When a short-term certificate is 5 months old, no rotation occurs.
-.. When a long-term certificate is 8.5 years old, no rotation occurs.
-
-. Yellow zone:
-.. When a short-term certificate is 8 months old, it is rotated when {product-title} starts or restarts.
-.. When a long-term certificate is 9 years old, it is rotated when {product-title} starts or restarts.
-
-. Red zone
-.. When a short-term certificate is 8 months old, {product-title} restarts to rotate and apply a new certificate.
-.. When a long-term certificate is 9 years old, {product-title} restarts to rotate and apply a new certificate.
+Certificates that are expired or close to their expiration dates need to be rotated to ensure continued {product-title} operation. When {product-title} restarts for any reason, certificates that are close to expiring are rotated. A certificate that is set to expire soon, or has expired, can cause an automatic {product-title} restart to perform a rotation.
 
 [NOTE]
 ====
 If the rotated certificate is a Certificate Authority, all of the certificates it signed rotate.
 ====
 
-.Stoplight timeline of {product-title} certificate validity.
-image::microshift-cert-rotation.png[<{product-title} graph with symbolic green-yellow-red stoplight map of certificates>]
+[id="microshift-st-certificate-rotation_{context}"]
+=== Short-term certificates
+The following situations describe {product-title} actions during short-term certificate lifetimes:
+
+. No rotation:
+.. When a short-term certificate is up to 5 months old, no rotation occurs.
+
+. Rotation at restart:
+.. When a short-term certificate is 5 to 8 months old, it is rotated when {product-title} starts or restarts.
+
+. Automatic restart for rotation:
+.. When a short-term certificate is more than 8 months old, {product-title} can automatically restart to rotate and apply a new certificate.
+
+.Graph of {product-title} short-term certificate validity.
+image::324_RHbM_Certificate_Rotation_0323_short-term.png[<{product-title} bar graph of certificate timelines>]
```

Comment:
This image makes it look like the cert might be regenerated on restart any time during the 8 months, but that's not true. If the middle bar only covered the time between 5 and 8 months, and did not go all the way to the left side of the diagram, it would more accurately reflect a phase change. Similarly, the bottom line for "Restart to rotation" should say "Restart to rotate" and should start at 8 and go to 10 months.

Alternatively, one bar showing the 3 phases in sequence would be even clearer.

Reply:
@dhellmann We are past the approval stage for these images. I can only remove them at this point.